web/client/codechecker_client/cmd/cmd.py

```diff
@@ -99,75 +101,90 @@ def get_argparser_ctor_args():
     }
 
 
-def __add_common_arguments(parser,
-                           needs_product_url=True,
-                           has_matrix_output=False,
-                           allow_html_output=False):
-    """
-    Add some common arguments, like server address and verbosity, to parser.
-    """
+def __add_output_formats(parser, output_formats=None,
+                         allow_multiple_outputs=False):
+    """ Add output arguments to the given parser. """
+    if not output_formats:
+        # Use default output formats.
+        output_formats = DEFAULT_OUTPUT_FORMATS
+
+    if allow_multiple_outputs:
+        parser.add_argument('-o', '--output',
+                            nargs="+",
+                            dest="output_format",
+                            required=False,
+                            default=["plaintext"],
+                            choices=output_formats,
+                            help="The output format to use in showing the "
```

Comment:
We could indicate if multiple formats are allowed.
Either by using `format(s)`, or mentioning that `multiple formats can be given` in the help-text.